### DISCUSSION TITLE
Support for arm based Mac and Windows systems. VMs don't work without it.

### DISCUSSION THREAD

**Discussion by eprisencc** [elementary/triage#625] [Ideas]
Support for arm based Mac and Windows systems.  VMs don't work without it.

**Comment by ryonakano** [comment]
Maybe duplicate of https://github.com/elementary/os/issues/682?

**Comment by cyrneko** [comment]
There are now ARM64 EFI images available in early access at https://builds.elementary.io

You can get access to that by [sponsoring Elementary](https://github.com/sponsors/elementary) at 1 USD or more

**Comment by ryonakano** [comment]
Closing as resolved since Early Access images are now available as @cyrneko mentioned.
